src/main/java/thePackmaster/cards/marisapack/MasterSpark.java

```diff
@@ -1,46 +1,82 @@
 package thePackmaster.cards.marisapack;
 
 import com.megacrit.cardcrawl.actions.AbstractGameAction;
-import com.megacrit.cardcrawl.cards.DamageInfo;
 import com.megacrit.cardcrawl.characters.AbstractPlayer;
 import com.megacrit.cardcrawl.monsters.AbstractMonster;
 import com.megacrit.cardcrawl.vfx.combat.MindblastEffect;
 import thePackmaster.util.Wiz;
 
 import static thePackmaster.SpireAnniversary5Mod.makeID;
-import static thePackmaster.util.Wiz.doDmg;
 
-public class MasterSpark extends AbstractMarisaCard implements AmplifyCard{
+public class MasterSpark extends AbstractMarisaCard implements AmplifyCard {
     public final static String ID = makeID(MasterSpark.class.getSimpleName());
-    private static final int DMG = 9, UPG_DMG = 1;
-    private static final int MAGIC = 17, UPG_MAGIC = 3;
+    private static final int DMG = 8, UPG_DMG = 2;
+    private static final int MAGIC = 17 - DMG, UPG_MAGIC = 23 - (DMG + UPG_DMG);
 
     public MasterSpark() {
-        super(ID, 1, CardType.ATTACK, CardRarity.COMMON, CardTarget.ENEMY);
+        super(ID, 1, CardType.ATTACK, CardRarity.COMMON, CardTarget.ALL_ENEMY);
         damage = baseDamage = DMG;
-        baseSecondDamage = secondDamage = MAGIC;
-
+        baseMagicNumber = magicNumber = DMG + MAGIC;
+        isMultiDamage = true;
     }
 
     public void use(AbstractPlayer p, AbstractMonster m) {
         Wiz.vfx(new MindblastEffect(p.hb.cX, p.hb.cY, p.flipHorizontal), 0.1F);
-        dmg(m, AbstractGameAction.AttackEffect.NONE);
+        int tmpBaseDamage = baseDamage;
+        if (isAmplified(this)) {
+            baseDamage = baseMagicNumber;
+            calculateCardDamage(null);
+        }
+        allDmg(AbstractGameAction.AttackEffect.NONE);
+        if (isAmplified(this)) {
+            baseDamage = tmpBaseDamage;
+            applyPowers();
+        }
+    }
+
+    // lots of jank here because using secondDamage will make the multiDamage array not upgrade
```

Comment:
To be fair, this is similar to what the base game does for cards like Halt. Which I agree is a bit jank, but it's jank with base game precedent that reliably works, which is something I'm on board with.